group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/StreamsGroup.java

```diff
@@ -1004,7 +1004,7 @@ private void removeTaskProcessIdsFromSet(
      *
      * @param tasks     The assigned tasks.
      * @param processId The process ID.
-     * @throws IllegalStateException if the partition already has an epoch assigned. package-private for testing.
+     * @throws IllegalStateException if the existing partition has larger epoch than the new one. package-private for testing.
```

Comment:
I think we don't compare epochs for streams now?
```suggestion
     * package-private for testing.
```